pkg/token/execCredentialPlugin.go

```diff
@@ -28,6 +32,16 @@ type execCredentialPlugin struct {
 }
 
 func New(o *Options) (ExecCredentialPlugin, error) {
+	env := os.Getenv(execInfoEnv)
+	fmt.Fprintln(os.Stderr, os.Getenv(execInfoEnv))
+	var execCredential clientauthentication.ExecCredential
+	error := json.Unmarshal([]byte(env), &execCredential)
```

Comment:
a more idiomatic way is
if err := ...; err != nil {
}

Reply:
Changed per comments and please check next commit. Thanks!